src/core/qgsrendercontext.h

```diff
@@ -1024,9 +1040,12 @@ class CORE_EXPORT QgsRenderContext : public QgsTemporalRangeObject
 
     double mSymbologyReferenceScale = -1;
 
-    //! Newer labeling engine implementation (can be NULLPTR)
+    //! Labeling engine implementation (can be NULLPTR)
     QgsLabelingEngine *mLabelingEngine = nullptr;
 
+    //! Label sink (can be NULLPTR)
```

Comment:
Does this docstring add any value? I would just drop it and let people refer to the nicely written docstring on the getter/setter.

Reply:
Not needed, but matches all other variable members in the class.